frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -420,22 +420,25 @@ export class ArchivedItemDetail extends TailwindElement {
     if (!this.crawl)
       return html`<sl-skeleton class="inline-block h-8 w-60"></sl-skeleton>`;
 
-    if (this.crawl.name) return this.crawl.name;
+    if (this.crawl.name)
+      return html`<span class="block truncate">${this.crawl.name}</span>`;
     if (!this.crawl.firstSeed || !this.crawl.seedCount) return this.crawl.id;
     const remainder = this.crawl.seedCount - 1;
-    let crawlName: TemplateResult = html`<span class="truncate"
+    let crawlName: TemplateResult = html`<span class="block truncate"
```

Comment:
Why do these need to be `block`?

Reply:
If they're not, truncation doesn't seem to work?  Will double check, but pretty sure it's to do with them not being cut off properly when overflowing as in-line elements.

Reply:
Nope, looks like I'm wrong!  Removed.  Good catch!! :D